staging/src/kubevirt.io/client-go/generated/kubevirt/clientset/versioned/typed/core/v1/streamer.go

```diff
@@ -1,4 +1,23 @@
-package kubecli
+/*
+ * This file is part of the KubeVirt project
```

Comment:
I think the streamer and websocket related logic could be separate package but I am not holding on this.

Reply:
As per the offline discussion, let's address this as a follow-up PR. Thank you!